setup.cfg

```diff
@@ -1,6 +1,3 @@
 [metadata]
 # Include the license file in wheels.
 license_file = license.txt
-
-[bdist_wheel]
-universal = 1
```

Comment:
What's the right thing to put here then? I'd still like to build a py3 wheel, not a version-specific one.

Reply:
wheel should do the right thing with pure-Python packages and tag this as py3.

Even if it was tagged as py35, a pure-Python wheel is forwards-compatible within a major version, I believe. black is distributed as a py36 wheel which still gets installed by pip on Python 3.7, for example.

Reply:
Yep, that seems to be what it does on my machine. Thanks!